### PR TITLE
feat: update aidant connect sections

### PR DESCRIPTION
**Pull request by rdubigny**

### CODE REVIEW

**Comment by rdubigny** [comment]
[Mise à jour formulaire Aidant Connect](https://trello.com/c/MGMovTxk/814-mise-%C3%A0-jour-formulaire-aidant-connect)

File: app/models/enrollment.rb [hunk 1]
```diff
@@ -167,41 +167,15 @@ def clean_and_format_scopes
...
-      additional_content["information_agreement"] =
-        additional_content["information_agreement"].to_s == "true"
+    self.additional_content = additional_content.transform_values do |v|
+      case v.to_s
+      when "true"
+        true
+      when "false"
+        false
+      else
+        v
```
Comment (XavierJp) [inline app/models/enrollment.rb:177]: v ?

File: app/models/document/delegation_service_public.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class Document::DelegationServicePublic < Document
+end
```
Comment (XavierJp) [inline app/models/document/delegation_service_public.rb:2]: Is this intentional to have this empty file ?

File: app/models/enrollment.rb (hunk 1 above)
Comment (XavierJp) [reply app/models/enrollment.rb:177]: maybe could we use here a more expressive variable name ?

File: app/models/document/delegation_service_public.rb (hunk 2 above)
Comment (rdubigny) [reply app/models/document/delegation_service_public.rb:2]: ruby « magic » !

**Pull request merged**